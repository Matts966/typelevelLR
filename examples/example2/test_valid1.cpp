
#include <iostream>
#include "example2.hpp"
using namespace example2;

int main() {
  std::cout << *begin()->num( 42 )->end() << std::endl;
  return 0;
}
