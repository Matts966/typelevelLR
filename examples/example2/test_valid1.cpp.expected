TToE(FToT(Num(42)))